app/config/sources.inc.php

```diff
@@ -111,6 +111,10 @@
         'deadline'          => '2016-03-15',
         'supported_locales' => $mozillaorg,
     ],
+    'firefox/all.lang' => [
+        'deadline'          => '2017-04-04',
+        'supported_locales' => $mozillaorg,
```

Comment:
We should probably use `$firefox_desktop_android` here, the same we use for firefox/new

We should also change priority: not sure if 1 or 2, definitely not 3 (default for mozilla.org)